Studs/StudsViewController.swift

```diff
@@ -16,15 +16,15 @@ final class StudsViewController: UITabBarController {
 
     private func setupApplication() {
         let eventsViewController = UINavigationController(rootViewController: EventsViewController.instance()).apply {
-            $0.tabBarItem = UITabBarItem(title: "Events", image: nil, tag: 0)
+            $0.tabBarItem = UITabBarItem(title: "Events", image: UIImage(named: "eventsTab"), tag: 0)
```

Comment:
A future tip is to use imageLiteral. You get it just by typing the image resource's name :)